tests/queries/0_stateless/25402_show_columns.reference

```diff
@@ -0,0 +1,38 @@
+int32	Nullable(Int32)	1		\N	
+str	String	0	SOR	\N	
+uint64	UInt64	0	PRI SOR	\N	
```

Comment:
Please correct me if I am wrong: does it mean that SHOW COLUMNS via MySQL wire protocol will return ClickHouse types in the response without translation to MySQL types?

Reply:
To start with some nitpicking (sorry :-) ), let's not confuse the "MySQL wire protocol" with "MySQL SQL dialect". This PR is strictly about the latter.

Yes, we'll return ClickHouse types. Is that a problem? Afaik, no administrative SQL statement in ClickHouse returns the types of another DBMS (and there are many of them: MySQL, Postgres, sqlite, ...).

Reply:
@rschu1ze, thanks for the correction :) 
We will check how it behaves with tools such as Looker Studio, where they discover the database schema using the `SHOW COLUMNS` command.

Reply:
^ (edited)

Reply:
Sounds good. If we can make SHOW COLUMNS even more MySQL like (it already has some few MySQL-specific quirks, see the [docs](https://github.com/ClickHouse/ClickHouse/pull/48017/files#diff-1b1c5879701b0ee7a53f1a0fd2746d2d9a3793fedc0e0ef366ca107ec8116db9)), I'd be happy to implement it.